### PR TITLE
Remove stale docs: tcp reset and health protocol

### PR DESCRIPTION
**Pull request by MartinForReal**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
remove stale docs for the features which we no longer support: TCP reset and health-protocol 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: site/content/en/topics/loadbalancer.md [hunk 1]
```diff
@@ -162,9 +160,9 @@ Currently, the default protocol of the health probe varies among services with d
...
+Since v1.20, two service annotations `service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path` are introduced, which determine the new health probe behavior. If the spec.ports.appProtocol is set, both local and cluster TCP services would use the specified health probe protocol. If the `service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path` is set, the specified request path would be used instead of `/healthz`. Note that the request path would be ignored when using TCP or the spec.ports.appProtocol is empty. More specifically:
```
Comment (jwtty) [inline site/content/en/topics/loadbalancer.md:163]: Just one annotation?

Comment (MartinForReal) [reply site/content/en/topics/loadbalancer.md:163]: yes

**Pull request merged**